api/src/controllers/documentFields.js

```diff
@@ -44,7 +44,7 @@ const projectDocumentsController = function(router) {
   router.put(
     '/:id',
     validateParams(paramSchema),
-    validateBody(bodySchema),
+    // validateBody(bodySchema), // this shouldn't validate bodySchema
```

Comment:
this comment is inaccurate, the body schema (aka the payload to this endpoint) should definitely be validated. The problem is the body schema between the "create" and "update" endpoints should not the same (but currently is, as implemented). I updated this Asana task to reflect the update: https://app.asana.com/0/1103604360526176/1118485962871954

For now, let's update the comment.
```suggestion
    // validateBody(bodySchema), // TODO: update body schema for PUT request. All fields should not be required on PUT
```